sampleconfig/orderer.yaml

```diff
@@ -47,6 +47,10 @@ General:
         # Consensus messages are dropped if the buffer is full, and transaction
         # messages are waiting for space to be freed.
         SendBufferSize: 10
+        # InsecureSkipVerify will disable the checking on tls certificate chain and the hostname.
+        # Be careful to change to true, and only use this when doing test or the OSNs are behind
+        # network middle devices.
+        InsecureSkipVerify: false
```

Comment:
Even if we decide to support this, we shouldn't expose this anyway. We can just support it in the code but not expose this in the config sample.

Reply:
Sure, we can consider to hide this config item, as others.
But IMHO, we should allow users to have more flexibility to control the code they are using as OSS, instead of playing hiding and finding?

Reply:
We already have a bunch of [easter eggs](https://hyperledger-fabric.readthedocs.io/en/release-1.4/raft_configuration.html#local-configuration) in the Raft config, give or take :)